### ISSUE TITLE
jQuery polluting console

### ISSUE THREAD

**Issue by jonluca** [extesy/hoverzoom#340]
**ATTENTION!** If you don't follow this template then your issue will be immediately closed.

## Description of the problem

Hoverzoom is currently polluting the console with [Violation]'s from jQuery. [See here](https://i.imgur.com/YDUI4RN.png)

## Url of the page where the problem occurs (not the image url)

All pages

## Steps to reproduce the problem (include image url here)
1. Open developer tools, navigate to a page and look at console.

## OS version, extension version (Chrome only, Firefox is not supported)
macos, HZ+ 1.0.125, Chrome Version 64.0.3282.167 (Official Build) (64-bit)

**Comment by extesy** [comment]
@jonluca Can't reproduce on github.com and reddit.com, so your "all pages" claim in invalid. Please provide page url.

**Comment by jonluca** [comment]
Odd, it happens on github and reddit. See [here](https://i.imgur.com/L2DUkFA.png)

**Comment by jonluca** [comment]
[And the call stack](https://i.imgur.com/2utzF7d.png)

**Comment by extesy** [comment]
Doesn't happen for me at the moment. I'll keep an eye on this but it might be a conflict with some other extension you have. Thanks for the call stack, I'll see if it gives me any clues.

**Comment by jonluca** [comment]
Yeah no worries! Thanks for looking into it. I'll try disabling some other extensions and checking why it's happening.

**Comment by jonluca** [comment]
Still happens for me when it's the only extension enabled. I disabled everything except HZ+ and [it looks like this](https://i.imgur.com/rZXcSLi.png)

**Comment by extesy** [comment]
Apparently it's a feature of DevTools which was added a few months ago: https://codereview.chromium.org/2863183003/. I couldn't find a toggle to enable or disable it, but seems like the intent is to display a legitimate warning. So if you could provide a stack trace for the second warning as well that would be helpful.

**Comment by jonluca** [comment]
Interesting. It happens in the bindEvents() function, around line 999 of hoverzoom.js.

[Seen here](https://i.imgur.com/T4YkS0Q.png) - Line 1000 is the NodeInserted warning, and 1003 is the non-passive event listener warning.

**Comment by jonluca** [comment]
[This might help](https://www.sitepoint.com/html5-javascript-mouse-wheel/) - I think it currently uses the older `onmousewheel`, when it could just handle it with `mousewheel`

**Comment by extesy** [comment]
Passive event handler warning will be impossible to fix until https://github.com/jquery/jquery/issues/2871 is implemented
